src/core/modules/Nebula.cpp

```diff
@@ -887,6 +883,28 @@ void Nebula::renderMarkerRoundedRect(StelPainter& sPainter, const float x, const
 	sPainter.enableClientStates(false);
 }
 
+void Nebula::renderMarkerPointedCircle(StelPainter& sPainter, const float x, const float y,
+									   float size, const Vec3f color, const bool insideRect) const
```

Comment:
I see the intent of this "insideRect", but where is it used? Maybe a line like 
```
if (insideRect)
    size*=0.95;
```
is still missing? (The scaling value may be something else than 0.95...)

Reply:
You're right, I forgot to put an if before line 899.